src/dotnet/Fable.Compiler/Transforms/FSharp2Fable.Util.fs

```diff
@@ -182,8 +182,7 @@ module Helpers =
 
     let unionCaseTag (ent: FSharpEntity) (unionCase: FSharpUnionCase) =
         try
-            let name = unionCase.Name
-            ent.UnionCases |> Seq.findIndex (fun uci -> name = uci.Name)
+            ent.UnionCases |> Seq.findIndex (fun uci -> unionCase.Name = uci.Name)
```

Comment:
Will this be OK? I'm always afraid that a property in FCS may evaluate more code than expected, that's why I was caching the name here so the `.Name` property is not evaluated multiple times.

Reply:
Benchmark show no degradation, in fact it's a bit less allocation.